templates/config.edn

```diff
@@ -146,33 +169,38 @@
  ;;  :editor/right           ["ctrl+l" "right"]}
  :shortcuts {}
 
- ;; By default, pressing `Enter` in the document mode will create a new line.
- ;; Set this to `true` so that it's the same behaviour as the usual outliner mode.
- :shortcut/doc-mode-enter-for-new-block? false
+ ;; Configure the behavior of pressing Enter in document mode.
+ ;; if set to true, pressing Enter will create a new block.
+ ;; Default value: true
+ :shortcut/doc-mode-enter-for-new-block? true
 
- ;; Block content larger than `block/content-max-length` will not be searchable
- ;; or editable for performance.
+ ;; Limit the searchable and editable block content length for performance
+ ;; Default value: 10000
  :block/content-max-length 10000
 
- ;; Whether to show command doc on hover
+ ;; Display command documentation on hover.
+ ;; Default value: true
  :ui/show-command-doc? true
 
- ;; Whether to show empty bullets for non-document mode (the default mode)
+ ;; Display empty bullet points.
  :ui/show-empty-bullets? false
 
- ;; Pre-defined :view function to use with advanced queries
+ ;; Advanced queries view function.
+ ;; Format the query result with `pprint`. 
+ ;; 
```

Comment:
```suggestion
 ;; Advanced queries view function.
 ;; Format the query result with `pprint`. 
```

Reply:
Is "Format the query result with `pprint`." describing the default setting value shown below, or the setting in general? Unclear.